doc/bc/changes-5.4.md

```diff
@@ -189,6 +189,18 @@ Changes affecting version compatibility with former or future versions.
   when deleting last version of the Content. Since Content without a version does not make sense, in
   this case `eZ\Publish\Core\Repository\ContentService::deleteContent()` should be used instead.
 
+* 5.4.5: $fieldFilters argument on Search service has been renamed to $languageFilter
+  Reason is to better communicate what the argument is used for. No changes
+  to it's structure is done so this is a purly cosmetic change.
+
+* 5.4.5: $languageFilter behaviour has changed to work as priority list of languages
```

Comment:
Unless will implement it for Legacy engine as well, this is valid only for Solr ATM.